config/auth0.php

```diff
@@ -6,30 +6,33 @@
  * Please review available configuration options here:
  * https://github.com/auth0/auth0-PHP#configuration-options.
  */
+$isKeycloakEnabled = env('IS_KEYCLOAK_ENABLED', false);
+
 return [
-    'enabled' => env('AUTH0_CLIENT_ENABLED', true),
+    // You can always login, either with auth0 or Keycloak
+    'enabled' => true,
 
     // Should be assigned either 'api', 'management', or 'webapp' to indicate your application's use case for the SDK.
     // Determines what configuration options will be required.
     'strategy' => env('AUTH0_LOGIN_STRATEGY', \Auth0\SDK\Configuration\SdkConfiguration::STRATEGY_REGULAR),
 
     // Auth0 domain for your tenant, found in your Auth0 Application settings.
-    'domain' => env('AUTH0_LOGIN_DOMAIN'),
+    'domain' => $isKeycloakEnabled ? env('KEYCLOAK_LOGIN_DOMAIN') : env('AUTH0_LOGIN_DOMAIN'),
 
     // Auth0 Management API domain for your tenant, found in your Auth0 Application settings.
-    'managementDomain' => env('AUTH0_LOGIN_MANAGEMENT_DOMAIN'),
+    'managementDomain' => $isKeycloakEnabled ? env('KEYCLOAK_LOGIN_MANAGEMENT_DOMAIN') : env('AUTH0_LOGIN_MANAGEMENT_DOMAIN'),
```

Comment:
If Keycloak is enabled will this not break the creation of keys on Auth0? Or is this management part only used to get users?